tools/supportedVersions.linq

```diff
@@ -0,0 +1,71 @@
+<Query Kind="Program">
+  <NuGetReference>NuGet.Core</NuGetReference>
+  <Namespace>NuGet</Namespace>
+  <Namespace>System.Collections.Concurrent</Namespace>
+  <Namespace>System.Threading.Tasks</Namespace>
+</Query>
+
+void Main()
+{
+	var corePackageName = "NServiceBus";
+	var nuGet = PackageRepositoryFactory.Default.CreateRepository("https://www.nuget.org/api/v2/");
+	var minVersion = new SemanticVersion("3.3");
+	var majorSupportDurationInYears = 3;
+
+	var fileLocation = Path.Combine(Util.CurrentQuery.Location, @"..\nservicebus\upgrades\support-dates.include.md");
+
+	var releasesByMajor =
+		(from package in nuGet.FindPackagesById(corePackageName)
+		 where package.IsListed()
+		 where string.IsNullOrWhiteSpace(package.Version.SpecialVersion)
+		 where package.Version >= minVersion
+		 let result = new
+		 {
+		 	package.Version, 
+			package.Published, 
+			VersionString = package.Version.ToString()
+		 }
+		 group result by result.Version.Version.Major into g
+		 select g).Dump("All Versions by Major", 2);
+		 
+	var majorDates = 
+		(from majorRelease in releasesByMajor
+		 let initialRelease = majorRelease.Min(x => x.Published)
+		 let latestMinor = majorRelease.Max(r => r.Published)
```

Comment:
I'd say if we only show majors the minor data isn't adding any value.

Wouldn't it be relevant to show the minor(s) that we support currently on this page as well?